test/suite/helper_tiered.py

```diff
@@ -59,7 +63,7 @@ def get_auth_token(storage_source):
 def get_bucket_name(storage_source, i):
     if storage_source == 's3_store':
         return s3_buckets[i]
-    if storage_source == 'dir_store':
+    else:
```

Comment:
revert this please.

Reply:
addressed https://github.com/wiredtiger/wiredtiger/pull/8707/commits/61b13a704e7d58933ead288cc550b6b4492ad90c